release-1.8/release_notes_draft.md

```diff
@@ -50,6 +50,20 @@ promotion indicates RBAC's production readiness and adoption. Today, the
 authorizer is turned on by default by many distributions of Kubernetes, and is a
 fundamental aspect of a secure cluster.
 
+### SIG Node
+
+[SIG Node][] is responsible for the components which support the controlled 
+interactions between pods and host resources as well as managing the lifecycle
+of pods scheduled on a node. For the 1.8 release SIG node was focused on
```

Comment:
i would just say the following:

sig-node iterated on its ability to run more types of workloads optimally, while making incremental improvements to overall node reliability